nixos/modules/security/polkit.nix

```diff
@@ -14,13 +14,16 @@ in
 
     security.polkit.enable = mkEnableOption (lib.mdDoc "polkit");
 
+    security.polkit.debug = mkEnableOption (lib.mdDoc "debug logs from polkit. This is required in order to see log messages from rule definitions.");
```

Comment:
Shouldn't this be false by default?

Reply:
Isn't the default here false ? https://github.com/NixOS/nixpkgs/blob/master/lib/options.nix#L95

Reply:
yeah right